src/test/java/AdvancementAPITest.java

```diff
@@ -38,8 +39,14 @@ public void setUp() {
     @Test
     public void createAndSave() {
 
+
+        TextComponent textComponent = new TextComponent("Addiction!");
+        textComponent.setBold(true);
+        textComponent.setColor(ChatColor.GOLD);
+
+
         AdvancementAPI advancementAPI = AdvancementAPI.builder(new NamespacedKey("test", "addiction"))
-                .title(new TextComponent("Addiction!"))
+                .title(textComponent)
                 .description(new TextComponent("Eat an Apple"))
                 .icon("minecraft:golden_apple")
                 .trigger(AdvancementAPI.Trigger.builder(AdvancementAPI.Trigger.TriggerType.CONSUME_ITEM, "test").condition(AdvancementAPI.Condition.builder("potion", new ItemStack(Material.APPLE, 1))))
```

Comment:
Good implementation of the builder pattern here. Make sure to also have another test comparing two strings with an assertion. One made by a human and one made by the class to ensure the API works correctly. Make sure to trim both upon doing so.